apps/xboxplaydates/xboxplaydates.star

```diff
@@ -0,0 +1,65 @@
+"""
+Applet: XboxPlaydates
+Summary: Quotes from Xbox Playdates
+Description: Displays quotes from the Xbox Playdates Teams.  You can find out more about Xbox Playdates at XboxPlaydates.org .
+Author: PalmettoBling
+"""
+
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+
+DEFAULT_WHO = "world"
+QUOTEURL = "https://www.xboxplaydates.us/playdatesQuotes/randomquote"
+
+def main():
+    rep = http.get(QUOTEURL)
+    if rep.status_code != 200:
+        fail("Something happened trying to get the quote. %d", rep.status_code)
+
+    quoteId = rep.json()["id"]
+    quote = rep.json()["quote"]
+    attribution = rep.json()["attribution"]
+
+    return render.Root(
+        child = render.Column(
+            expanded = False,
+            children = [
+                render.Text(
+                    content = ("# %s: " % quoteId),
+                    color = "#099",
+                ),
+                render.Marquee(
+                    width = 64,
+                    height = 12,
+                    offset_start = 16,
+                    child = render.WrappedText(
+                        height = 16,
+                        linespacing = -1,
+                        content = ("%s" % quote),
+                    ),
+                ),
+                render.Marquee(
+                    width = 64,
+                    offset_end = 0,
+                    child = render.Text(
+                        color = "#0a0",
+                        content = ("-%s" % attribution),
+                    ),
+                ),
+            ],
+        ),
+    )
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
```

Comment:
This fields seems to be unused.

Reply:
You are correct.  Thank you for your time and review.  Removed the function and library call.  Artifact left from app file generation process.